aws-android-sdk-auth-google/build.gradle

```diff
@@ -17,9 +17,8 @@ android {
 
 dependencies {
     api project(':aws-android-sdk-auth-core')
-    api 'com.google.android.gms:play-services-auth:9.8.0'
-    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-
-    testImplementation 'junit:junit:4.12'
+    api 'com.google.android.gms:play-services-auth:18.0.0'
```

Comment:
this looks like a huge version jump. this didn't break anything?

Reply:
Not at compile-time, anyway. We don't have good test coverage on this module, so it's difficult to know what if any runtime issues may have been resolved / created.